dev-util/coverity/coverity-2017.07.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Coverity provides the results of analysis on open source coding projects"
+HOMEPAGE="https://www.coverity.com"
+SRC_URI="amd64? ( cov-analysis-linux64-${PV}.tar.gz )
+	x86? ( cov-analysis-linux-${PV}.tar.gz )"
+
+LICENSE="coverity-PLA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+L10N="ja ko zh-CN"
+for _l in ${L10N} ; do
+	IUSE="${IUSE} l10n_${_l}"
+done
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+RESTRICT="fetch mirror bindist"
+
+QA_PREBUILT="opt/coverity/bin/*"
+
+pkg_nofetch() {
+	eerror "Please go to https://scan.coverity.com/download"
+	use amd64 && eerror "and download Linux64"
+	use x86   && eerror "and download Linux32"
+	eerror "then put the file in ${DISTDIR}"
+	use amd64 && eerror "named as cov-analysis-linux64-${PV}.tar.gz"
+	use x86   && eerror "named as cov-analysis-linux-${PV}.tar.gz"
+}
+
+src_unpack() {
+	default
+
+	use amd64 && S="${WORKDIR}/cov-analysis-linux64-${PV}"
+	use x86   && S="${WORKDIR}/cov-analysis-linux-${PV}"
+}
+
+src_prepare () {
+	default
+
+	echo "PATH='/opt/${PN}/bin'" >  "${T}/99${PN}"
+
+	# Rename invalid locale files
+	mv locale/zh-cn locale/zh-CN
+
+	# Cleanup
+	find -type f -name '*.DS_Store' -exec rm {} +
+	find -type f -name '*.bak' -exec rm {} +
+
+	rm -rf lib/python*
+
+	# Already available in /usr/portage/licenses
+	rm -r doc/*/licenses
+	rm    doc/*/ScanUserAgreementv2016.1.pdf
+}
+
+src_install () {
+	doenvd "${T}/99${PN}"
+
+	insinto "/opt/${PN}"
+	doins -r config dtd lib
+
+	into "/opt/${PN}"
+	dobin bin/cov-*
+	dobin bin/lib*.so
+
+	insinto "/opt/${PN}/doc"
+	doins -r doc/en
```

Comment:
Does the documentation have to be installed under `/opt` rather than the more typical `/usr/share/doc`?

Use `dodoc` to install documentation, and if you have to install it to an alternate location you can define it with `docinto`.

Reply:
Yes, the do unfortunately :/